cmd/kube-proxy/app/server_others.go

```diff
@@ -108,14 +108,7 @@ func newProxyServer(
 	iptInterface = utiliptables.New(execer, protocol)
 	kernelHandler = ipvs.NewLinuxKernelHandler()
 	ipsetInterface = utilipset.New(execer)
-	canUseIPVS, err := ipvs.CanUseIPVSProxier(kernelHandler, ipsetInterface)
```

Comment:
I think we should keep this call and pass `canUseIPVS` down to `getProxyMode` instead.

Reply:
done, PTAL @andrewsykim